src/components/Layout.tsx

```diff
@@ -78,6 +78,9 @@ const globalStyles = css({
     WebkitFontSmoothing: 'auto',
     ...type.features.addSwashes,
   },
+  'h1 .anchor svg, h2 .anchor svg, h3 .anchor svg, h4 .anchor svg, h5 .anchor svg, h6 .anchor svg' : {
+    fill: variables.colors.text.link,
```

Comment:
```suggestion
    fill: variables.colors.text.primary,
```

What do you think about using the text color? That was the intention with light mode:

<img width="584" alt="image" src="https://user-images.githubusercontent.com/3277153/72688447-3ffc8d80-3abc-11ea-9e78-5f74ec49ad4d.png">

Reply:
Indeed!